charts/devtron/templates/workflow.yaml

```diff
@@ -1184,7 +1184,7 @@ spec:
         - workflow-controller-configmap
         - --executor-image
         {{- if $.Values.workflowController.IMDSv2Enforced }}
-        - {{ $.Values.workflowController.executorImage }}
+        - {{ include "common.image" (dict "component" $.Values.workflowController "global" $.Values.global "extraImage" $.Values.workflowController.executorImage  ) }}
         {{- else }}
         - quay.io/argoproj/argoexec:v3.0.7
```

Comment:
image is still here which should have been moved to values.yaml

Reply:
Done